include/gemmini.h

```diff
@@ -738,19 +611,21 @@ static void tiled_matmul_ws(size_t dim_I, size_t dim_J, size_t dim_K,
 */
```

Comment:
Nit: Remove the commented `tiled_matmul_ws`?

Reply:
@hngenc , do you need this for any reason?

Reply:
Keep the comments for now, please